### PR TITLE
Migrating the rules from Aqua commercial/Kube Enforcer into defsec

### PR DESCRIPTION
**Pull request by dheerajkadri**
Few rules which are present in Aqua product (which is used by Kube-enforcer) is not present in defsec. Migrating few policies to aquacommercial folder

### CODE REVIEW

File: internal/rules/policies/kubernetes/policies/aquacommercial/configMap_with_sensitive_test.rego [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package builtin.kubernetes.KSV01010
+
+test_configMap_with_sensitive_denied {
```
Comment (dheerajkadri) [inline internal/rules/policies/kubernetes/policies/aquacommercial/configMap_with_sensitive_test.rego:3]: Example for CM with sensitive data . Sensitive data is present in keys (**ex:email.id**) and also in key part of value (**username**=myusername)
image

File: internal/rules/policies/kubernetes/policies/aquacommercial/service_with_externalip_test.rego [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package builtin.kubernetes.KSV0108
+
+test_service_with_externalip_denied {
+	r := deny with input as {
```
Comment (dheerajkadri) [inline internal/rules/policies/kubernetes/policies/aquacommercial/service_with_externalip_test.rego:4]: Example for service with external ip
image

File: internal/rules/policies/kubernetes/policies/aquacommercial/service_with_externalip.rego [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+# failExtIpsOrName is true if service has external Name
+failExtIpsOrName {
+	kubernetes.kind == "Service"
+	not allowedNames[kubernetes.object.spec.externalName]
+}
+
+deny[res] {
+	failExtIpsOrName
+	msg := sprintf("%s %s in %s namespace should not set external IPs or external Name", [lower(kubernetes.kind), kubernetes.name, kubernetes.namespace])
```
Comment (owenrumney) [inline internal/rules/policies/kubernetes/policies/aquacommercial/service_with_externalip.rego:40]: Think this should be wrapped in a `kubernetes.format` like https://github.com/aquasecurity/defsec/blob/master/internal/rules/policies/kubernetes/policies/general/memory_not_limited.rego#L40

**Review by owenrumney — CHANGES_REQUESTED** [review]
Some small points, but otherwise looks good

**Review by owenrumney — APPROVED** [review]
LGTM

**Review by owenrumney — APPROVED** [review]
Nice! LGTM :rocket:

**Comment by simar7** [comment]
hi @dheerajkadri - just wondering how did you add the impact to the AVD markdown files when making this change? Was it manually added or the `make docs` makefile directive generate the markdowns for you?

**Pull request merged**